### PR TITLE
docs: add a domument for simulate process faults

### PR DESCRIPTION
**Pull request by en-jin19**
Signed-off-by: en-jin19 <jinenwei@pingcap.com>

In this PR, I have added a document about simulating process faults

This PR is translated from: chaos-mesh/website-zh#52

### CODE REVIEW

File: docs/simulate-process-chaos-in-physical-nodes.md [hunk 1]
```diff
@@ -1,6 +1,129 @@
...
+
+Global Flags:
+      --log-level string   the log level of chaosd, the value can be 'debug', 'info', 'warn' and 'error'
+
+Use "chaosd attack process [command] --help" for more information about a command.
+```
+
+Currently, Chaosd supports simulating that a process is killed or stopped.
+
+### Simulating that a process is killed
```
Comment (en-jin19) [inline docs/simulate-process-chaos-in-physical-nodes.md:39]: 这句作为小标题可能会有些长，是否可以翻译为 “Killing a process" 呢? @WangXiangUSTC

File: docs/simulate-process-chaos-in-physical-nodes.md [hunk 2]
```diff
@@ -1,6 +1,129 @@
...
+```
+
+The result is as follows:
+
+```bash
+chaosd attack process kill -p python
+Attack process python successfully, uid: 10e633ac-0a37-41ba-8b4a-cd5ab92099f9
+```
+
+### Simulating that a process is stopped
```
Comment (en-jin19) [inline docs/simulate-process-chaos-in-physical-nodes.md:84]: 这句作为小标题可能会有些长，是否可以翻译为 “Stopping a process" 呢? @WangXiangUSTC

File: docs/simulate-process-chaos-in-physical-nodes.md [hunk 3]
```diff
@@ -1,6 +1,129 @@
 ---
-title: 模拟进程故障
-sidebar_label: 模拟进程故障
+title: Simulate Process Faults
+sidebar_label: Simulate Process Faults
 ---
 
-## TODO
+This document describes how to use Chaosd to simulate process faults.The process faults use the Golang interface of the kill command to simulate the scenarios that the process is killed or stoped. You can create experiments either in command-line mode or service mode.
```
Comment (WangXiangUSTC) [inline docs/simulate-process-chaos-in-physical-nodes.md:6]:
```suggestion
This document describes how to use Chaosd to simulate process faults. The process faults use the Golang interface of the kill command to simulate the scenarios that the process is killed or stoped. You can create experiments either in command-line mode or service mode.
```

File: docs/simulate-process-chaos-in-physical-nodes.md (hunk 1 above)
Comment (WangXiangUSTC) [reply docs/simulate-process-chaos-in-physical-nodes.md:39]: 可以的

File: docs/simulate-process-chaos-in-physical-nodes.md [hunk 4]
```diff
@@ -1,6 +1,129 @@
...
+
+| Configuration Item | Abbreviation | Descpription                                                  | Value                                                                                     |
+|:------------------ |:------------ |:------------------------------------------------------------- |:----------------------------------------------------------------------------------------- |
+| process            | p            | The name or identifier of the process that needs to be killed | string; the default value is "".                                                          |
+| signal             | s            | The provided value of the process signal                      | int; the default value is 9. Currently, only SIGKILL, SIGTERM, and SIGSTOP are supported. |
+
+#### Example simulating that a process is killed
+
+```bash
+track process kiln python
```
Comment (WangXiangUSTC) [inline docs/simulate-process-chaos-in-physical-nodes.md:74]: 这里的命令不对

File: docs/simulate-process-chaos-in-physical-nodes.md (hunk 2 above)
Comment (WangXiangUSTC) [reply docs/simulate-process-chaos-in-physical-nodes.md:84]: 可以

**Comment by en-jin19** [comment]
I have revised the translations according to the review, please kindly take a look, thanks! @WangXiangUSTC

**Review by WangXiangUSTC — APPROVED** [review]
LGTM

**Pull request merged**